tests/integration/test_data/gomod_packages.yaml

```diff
@@ -5,7 +5,7 @@
 # expected_files: Expected source files <relative_path>: <file_URL>
 # expected_deps_files: Expected dependencies files (empty)
 # response_expectations: Parts of the Cachito response to check
-# purl: PURL of the package
+# content_manifest: PURLs for image contents part
```

Comment:
If possible, could we have 2 commits? 1 for this change, 1 for adding the new test

Reply:
Good idea, thank you!